xbmc/dialogs/GUIDialogYesNo.cpp

```diff
@@ -134,6 +147,35 @@ bool CGUIDialogYesNo::ShowAndGetInput(CVariant heading, CVariant text, bool &bCa
   return (dialog->IsConfirmed()) ? true : false;
 }
 
+int CGUIDialogYesNo::ShowAndGetInput(CVariant heading, CVariant text, CVariant noLabel, CVariant yesLabel, CVariant customLabel, unsigned int autoCloseTime)
+{
```

Comment:
Just a note, this duplicates code. Yes, the other implementations also do this. I would like to look at it when it's merged and try to combine it. Just a heads up.